src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Summary of how GitHub Issues and Pull Requests are processed
+---
+
+# Summary of how GitHub Issues and Pull Requests are processed
+
+GitHub is the main tool used by maintainers to manage the PrestaShop project.
+
+This page quickly describes how [Issues](https://guides.github.com/features/issues/) and [Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) are processed on GitHub.
+
+## The teams
+
+On GitHub, there are mainly three teams working together on the project.
+
+- Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers and Wording Managers.
+- Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts and QA automation engineers.
+- Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.
+
+A maintainer will naturally belong to the Developers team but can also assist and work with the Product Team and QA Team.
+
+## Issues
+
+Currently, there are 4 different types of Issues submitted by users on GitHub.
+
+### Feature Request
+
+When user submits a Feature Request, it can be either a Product feature request (example: add a new Back office page) or a Technical feature request (example: support PostgreSQL).
```

Comment:
```suggestion
When a user submits a Feature Request, it can be either a Product feature request (example: add a new Back office page) or a Technical feature request (example: support PostgreSQL).
```